src/NpmUtilities.js

```diff
@@ -54,7 +60,12 @@ export default class NpmUtilities {
         // build command, arguments, and options
         const opts = NpmUtilities.getExecOpts(directory, config.registry);
         const args = ["install"];
-        const cmd = config.npmClient || "npm";
+        let cmd = config.npmClient || "npm";
+
+        if (npmGlobalStyle) {
+          cmd = "npm";
+          args.push("--global-style");
```

Comment:
An `NpmUtilities.installInDir()` test case with an assertion similar to [this one](https://github.com/lerna/lerna/blob/master/test/NpmUtilities.js#L277) would help keep our coverage up.

```js
const config = { npmClient: "yarn" };
const npmGlobalStyle = true;

NpmUtilities.installInDir(directory, dependencies, config, npmGlobalStyle, (err) => {
  // ...
  expect(ChildProcessUtilities.exec).lastCalledWith("npm", ["install", "--global-style"], {
    // ...
```